test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -68,6 +68,7 @@ class SolidityExecutionFramework: public solidity::test::ExecutionFramework
 protected:
 	solidity::frontend::CompilerStack m_compiler;
 	bool m_compileViaYul = false;
+	bool m_showMetadata = false;
```

Comment:
```suggestion
	bool m_showMetadata = solidity::test::CommonOptions::get().showMetadata;
```
should work as well without the duplication in the constructors... but I'm fine with as it is as well.